pkgs/applications/terminal-emulators/kitty/default.nix

```diff
@@ -36,15 +31,17 @@ buildPythonApplication rec {
     ncurses
     lcms2
   ] ++ lib.optionals stdenv.isDarwin [
-    Cocoa
-    CoreGraphics
-    Foundation
-    IOKit
-    Kernel
-    OpenGL
+    darwin.apple_sdk.frameworks.Cocoa
+    darwin.apple_sdk.frameworks.CoreGraphics
+    darwin.apple_sdk.frameworks.Foundation
+    darwin.apple_sdk.frameworks.IOKit
+    darwin.apple_sdk.frameworks.Kernel
+    darwin.apple_sdk.frameworks.OpenGL
```

Comment:
Please revert this. I am not sure how to properly handle the UserNotifications framework but this is also not the solution. Maybe we should just wait until the SDK is bumped.

Reply:
Done